exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/TestKit.java

```diff
@@ -448,39 +427,44 @@ public Builder withValidators(short validatorCount) {
     }
 
     /**
-     * Adds a service with which the TestKit would be instantiated. Several services can be added.
+     * Adds a service artifact which would be deployed by the TestKit. Several service artifacts
+     * can be added.
+     *
+     * <p>Note that the corresponding service instance with equal serviceArtifactId should be
+     * created with {@link #withService(ServiceArtifactId, String, int, Any)}.
      */
-    public Builder withService(Class<? extends ServiceModule> serviceModule) {
-      services.add(serviceModule);
+    public Builder withDeployedService(ServiceArtifactId serviceArtifactId, String artifactFilename) {
```

Comment:
`withDeployedArtifact` so that it does not appear like a `withService` flavour, which works with deployed services?